app/assets/javascripts/analytics.js

```diff
@@ -60,5 +60,21 @@ Blacklight.onLoad(function(){
   $('.iiif-dnd').on('dragstart', function(e) {
     var manifest = $(e.currentTarget).data().manifest;
     window._gaq.push(['_trackEvent', 'IIIF DnD', 'dragged', manifest]);
-  })
+  });
+
+  // Track external link clicks
+  $('a[href]:not([href^="/"],[href^="#"])').on('click', function(e) {
```

Comment:
I _think_ this should be the correct selector. I'm not sure if there are any other considerations we should make here?

Reply:
This will end up tracking any time we use a fully qualified URL?  I don't think we use that a lot, but just want to make sure I know the considerations we may have to take moving forward.